cfme/common/provider_views.py

```diff
@@ -114,6 +114,16 @@ def is_displayed(self):
                 self.navigation.currently_selected == ['Middleware', 'Providers'])
 
 
+class ContainerProviderDetailsView(ProviderDetailsView):
```

Comment:
**Required**

Keep the pluralization consistent here, `ContainerProvider(...)` here and `ContainersProvider(...)` class names below.

Reply:
:+1:

Reply:
Can you confirm that you audited for uses of the plural class names and updated any existing imports/uses of those names in the rest of the framework?